packages/sdk/src/controllers/BrandKitController.ts

```diff
@@ -0,0 +1,284 @@
+import { getEditorResponseData } from '../utils/EditorResponseData';
+import { CharacterStyleController } from './CharacterStyleController';
+import { ColorStyleController } from './ColorStyleController';
+import { FontConnectorController } from './FontConnectorController';
+import { FontController } from './FontController';
+import { MediaConnectorController } from './MediaConnectorController';
+import { ParagraphStyleController } from './ParagraphStyleController';
+import { UndoManagerController } from './UndoManagerController';
+
+/**
+ * The BrandKitController is responsible for all communication regarding Brand Kits.
+ * Methods inside this controller can be called by `window.SDK.brandKit.{method-name}`
+ */
+
+export class BrandKitController {
+    /**
+     * @ignore
+     */
+    #editorAPI: EditorAPI;
+
+    /**
+     * @ignore
+     */
+    constructor(editorAPI: EditorAPI, sdk: SDK) {
+        this.#editorAPI = editorAPI;
+        this.colorStyleController = new ColorStyleController(this.#editorAPI);
+        this.fontController = new FontController(this.#editorAPI);
+        this.fontConnectorController = new FontConnectorController(this.#editorAPI);
+        this.mediaController = new MediaConnectorController(this.#editorAPI);
+        this.paragraphStyleController = new ParagraphStyleController(this.#editorAPI);
+        this.characterStyleController = new CharacterStyleController(this.#editorAPI);
+
+        this.undoManagerController = new UndoManagerController(this.#editorAPI, sdk);
+    }
+
+    private colorStyleController: ColorStyleController;
+
+    private fontController: FontController;
+
+    private fontConnectorController: FontConnectorController;
+
+    private mediaController: MediaConnectorController;
+
+    private paragraphStyleController: ParagraphStyleController;
+
+    private characterStyleController: CharacterStyleController;
+
+    private undoManagerController: UndoManagerController;
+
+    /**
+     * @experimental This method returns the local brandkit
+     * @returns brandkit with all assigned resources
+     */
+    get = async () => {
+        const colorsPromise = this.colorStyleController.getAll();
+        const fontsPromise = this.fontController.getFontFamilies();
+        const characterStylesPromise = this.characterStyleController.getAll();
+        const paragraphStylesPromise = this.paragraphStyleController.getAll();
+
+        const [colors, fonts, paragraphStyles, characterStyles] = await Promise.all([
+            colorsPromise,
+            fontsPromise,
+            paragraphStylesPromise,
+            characterStylesPromise,
+        ]);
+
+        const studioBrandKit = {
+            brandKit: {
+                colors: colors.parsedData,
+                fonts: fonts.parsedData,
+                paragraphStyles: paragraphStyles.parsedData,
+                characterStyles: characterStyles.parsedData,
+            },
+        };
```

Comment:
It seems like you omitted properties from both `StudioBrandKit` and `APIBrandKit` models.